molecule/default/create.yml

```diff
@@ -0,0 +1,34 @@
+---
+- name: Create
+  hosts: localhost
+  connection: local
+  gather_facts: false
+  no_log: "{{ molecule_no_log }}"
+  tasks:
+
+    # TODO: Developer must implement and populate 'server' variable
+
+    - when: server.changed | default(false) | bool
+      block:
+        - name: Populate instance config dict
+          set_fact:
+            instance_conf_dict: {
+              'instance': "{{ }}",
```

Comment:
Are these `instance_conf_dict` boilplate for Ansible, or specific to our agent configuration?

Reply:
Everything under `molecule/` is boilerplate.

Reply:
Got it